tests/models/test_classification.py

```diff
@@ -79,104 +79,6 @@ def test_assign_to_genre_can_remove_genre(self):
 
 
 class TestGenre(DatabaseTest):
-    def test_full_table_cache(self):
```

Comment:
These were really cache tests, tacked onto the tests for genre. So I moved them to the fulltablecache test file.